dcos_test_utils/__init__.py

```diff
@@ -0,0 +1,14 @@
+__all__ = [
```

Comment:
`__all__` is only required when you do a import of the style

`from package import *`

- Do we ever plan to do that? 

In style guides, we say we should avoid this? So, it will be better if we have empty `__init__` file.

Reply:
This is used for allowing the following:
```
import dcos_test_utils

m = dcos_test_utils.marathon.Marathon()
j = dcos_test_utils.jobs.Jobs()
```

without this you need to do:
```
import dcos_test_utils.marathon
import dcos_test_utils.jobs

m = dcos_test_utils.marathon.Marathon()
j = dcos_test_utils.jobs.Jobs()
```

I find the former to be much more friendly to work with

Reply:
If you have an empty `__init__.py` you will still be able to the first thing.

Reply:
It will be like this.

* `from dcos_test_utils import marathon`

Reply:
* The disadvantages of adding `__all__` is automated refacorting becomes difficult. Because the module is a string in `__all__`.
* You will have to make sure to keep it update.
* When you write mock based unittests, you can accidentally mock the package level alias over the actual module itself.

Reply:
More discussion is here: https://stackoverflow.com/questions/44834/can-someone-explain-all-in-python 

To me, looks like doing this is 

```
import dcos_test_utils.marathon
import dcos_test_utils.jobs
``` 

Or doing 

```
from dcos_test_utils import marathon
```

OR 
```
from dcos_test_utils import marathon as marathon_test_api
```

is not all bad.

One less place to maintain (that is adding strings in __all__)

Reply:
OK, that's enough points to ignore the small added convenience for some people. Really, I would prefer that everyone do `from dcos_test_utils import marathon, jobs, ...` :)